templates/bridgeserver2.yaml

```diff
@@ -137,6 +152,57 @@ Parameters:
 Conditions:
   CreateDevResources: !Equals [ !Ref BridgeEnv, dev ]
 Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    Properties:
+      AvailabilityZones: !Ref DatabaseClusterAvailabilityZones
+      BackupRetentionPeriod: !Ref DatabaseBackupRetentionDays
+      DatabaseName: 'BridgeDB'
+      DBClusterIdentifier: !Join
```

Comment:
Typically, allowing CFN to manage resource names makes resources more maintainable.  I would recommend that unless there's a requirement to give it a specific name.

Reply:
I'm not sure what CloudFormation's default naming convention is, and I'm unable to find any documentation. Since we may need to interact with the database directly for dev-ops, I'd rather ensure that we name the database something that we can read. I believe we follow the same pattern for nearly every other resource in our CFN templates.